test/Twig/Tests/EnvironmentTest.php

```diff
@@ -145,10 +145,18 @@ public function testExtensionsAreNotInitializedWhenRenderingACompiledTemplate()
         // force compilation
         $twig = new Twig_Environment($loader = new Twig_Loader_Array(array('index' => '{{ foo }}')), $options);
         $cache = $twig->getCacheFilename('index');
-        if (!is_dir(dirname($cache))) {
-            mkdir(dirname($cache), 0777, true);
+        $dir = dirname($cache);
+        $created = false;
+        if (!is_dir($dir)) {
+            if (false === $created = @mkdir($dir, 0777, true)) {
+                $this->markTestSkipped(sprintf('Failed to create test cache directory at "%s".', $dir));
```

Comment:
If the test cache dir cannot be created, I want a hard failure, not skipped tests.